### ISSUE TITLE
HTTP API sends start command in old format

### ISSUE THREAD

**Issue by smn** [praekeltfoundation/vumi-go#309]
What's sent is:

``` javascript
{
  "worker_name": "http_api_application",
  "command": "start",
  "args": [],
  "kwargs": {
    "user_account_key": "XXXX",
    "conversation_key": "XXXX"
  }
}
```

Which results in:

```
2013-06-02 07:30:30+0000 [WorkerAMQClient,client] Unhandled Error
        Traceback (most recent call last):
          File "/var/praekelt/vumi-go/ve/lib/python2.6/site-packages/twisted/internet/defer.py", line 1101, in gotResult
            _inlineCallbacks(r, g, deferred)
          File "/var/praekelt/vumi-go/ve/lib/python2.6/site-packages/twisted/internet/defer.py", line 1045, in _inlineCallbacks
            result = g.send(result)
          File "/var/praekelt/vumi-go/ve/src/vumi/vumi/service.py", line 283, in read_messages
            yield self.consume(message)
          File "/var/praekelt/vumi-go/ve/lib/python2.6/site-packages/twisted/internet/defer.py", line 1187, in unwindGenerator
            return _inlineCallbacks(None, gen, Deferred())
        --- <exception caught here> ---
          File "/var/praekelt/vumi-go/ve/lib/python2.6/site-packages/twisted/internet/defer.py", line 1045, in _inlineCallbacks
            result = g.send(result)
          File "/var/praekelt/vumi-go/ve/src/vumi/vumi/service.py", line 302, in consume
            message.content.body))
          File "/var/praekelt/vumi-go/ve/src/vumi/vumi/service.py", line 335, in consume_message
            return self.callback(message)
          File "/var/praekelt/vumi-go/go/vumitools/app_worker.py", line 154, in consume_control_command
            return cmd_method(*args, **kwargs)
        exceptions.TypeError: process_command_start() takes exactly 6 non-keyword arguments (1 given)
```

**Comment by smn** [comment]
99aa545844d2e27ca2c5c25e978438db0a988cb8 needs to be done for the http api worker

**Comment by jerith** [comment]
This is fixed as in #318.
